### PR TITLE
Style the modal scrollbar

### PR DESCRIPTION
**Pull request by helenclx**

### CODE REVIEW

**Review by Shree-77 — APPROVED** [review]

File: Food-Recipe/styles.css [hunk 1]
```diff
@@ -267,14 +267,29 @@ button:hover {
...
+
+.modal::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+.modal::-webkit-scrollbar-thumb {
+  background-color: #b1b1b1;
+  border-radius: 20px;
+}
+.modal::-webkit-scrollbar-button {
```
Comment (andreimaier) [inline Food-Recipe/styles.css:289]: I added another rule to effectively add padding to the to and bottom. No overflow across all 3 browsers, but still some overflow on mobile in chrome
image

File: Food-Recipe/styles.css [hunk 2]
```diff
@@ -267,14 +267,29 @@ button:hover {
...
+.modal::-webkit-scrollbar {
+  width: 10px;
+}
+
+.modal::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+.modal::-webkit-scrollbar-thumb {
+  background-color: #b1b1b1;
```
Comment (andreimaier) [inline Food-Recipe/styles.css:286]: I changed from black to gray to make it pop up less

**Review by andreimaier — APPROVED** [review]

**Comment by andreimaier** [comment]
good work Helen! I only made 2 small changes. Thank you for fixing this!

File: Food-Recipe/styles.css [hunk 3]
```diff
@@ -266,10 +266,7 @@ button:hover {
 
   width: 95%;
   max-height: 90%;
-  overflow-y: scroll;
-  scrollbar-width: thin;
```
Comment (helenclx) [inline Food-Recipe/styles.css]: @andreimaier The  `scrollbar-width` and `scrollbar-color:` are used for styling the scrollbar for Firefox. By removing these two lines, the scrollbar's style is back to default when viewed on Firefox, so I'm going to re-add and commit to the main branch.

**Pull request merged**